### PR TITLE
send confirmation dialog box should display correct smallest denom

### PR DESCRIPTION
**Pull request by UdjinM6**

### CODE REVIEW

**Comment by SCDeveloper** [comment]
ACK - been running this fix in DarkSilk under test for a while now.

File: src/qt/sendcoinsdialog.cpp [hunk 1]
```diff
@@ -260,7 +262,7 @@ void SendCoinsDialog::on_sendButton_clicked()
         strFunds = tr("using") + " <b>" + tr("anonymous funds") + "</b>";
         QString strNearestAmount(
             BitcoinUnits::formatWithUnit(
-                model->getOptionsModel()->getDisplayUnit(), 0.1 * COIN));
+                model->getOptionsModel()->getDisplayUnit(), vecPrivateSendDenominations.back()));
```
Comment (tgflynn) [inline src/qt/sendcoinsdialog.cpp]: 0.1*COIN becomes 0.01*COIN + 10 ?

Comment (UdjinM6) [inline src/qt/sendcoinsdialog.cpp]: yes, the actual text is `(privatesend requires this amount to be rounded up to the nearest 0.01000010 tDASH)` (0.1 was inaccurate btw, should say 0.10000100 tDASH instead to be precise).

**Comment by tgflynn** [comment]
utACK

**Pull request merged**